wp-modules/app/package.json

```diff
@@ -11,8 +11,7 @@
   "devDependencies": {
     "@types/jest": "^29.5.0",
     "@types/react-test-renderer": "^17.0.2",
-    "@wordpress/scripts": "^26.8.0",
-    "postcss": "^8.3.5",
```

Comment:
`postcss` is a dependency of `@wordpress/scripts`, we don't need it as a top-level dependency.